src/pages/beheer/instellingen/index.tsx

```diff
@@ -34,12 +78,138 @@ const CPSettingsPage: NextPage = () => {
+							<p className="p-cp-settings__description">
+								{t(
+									'Personaliseer hoe jouw leeszaal in het aanbod mag verschijnen op het Leeszalen overzicht. Naast een standaard achtergrondkleur kan je ook een thematische achtergrond afbeelding instellen.'
+								)}
+							</p>
+							<div className="p-cp-settings__leeszaal-controls">
+								<CardImage
+									className="p-cp-settings__leeszaal-image"
+									color={newColor}
+									logo="/images/logo-shd--small.svg"
+									id="placeholder id"
+									name={'placeholder name' || ''}
+									// image="/images/bg-shd.png"
+									size="short"
+								/>
+								<div className="p-cp-settings__leeszaal-color-picker">
+									<p className="p-cp-settings__subtitle">
+										{t('Achtergrondkleur')}
+									</p>
+									<ColorPicker
+										color={newColor}
+										onChange={(color) => {
+											if (!savedColor) {
+												console.log(color);
+
+												setSavedColor(color);
+											}
+											setNewColor(color);
+										}}
+									/>
+								</div>
+								<div className="p-cp-settings__leeszaal-image-controls">
+									<p className="p-cp-settings__subtitle">
+										{t('Achtergrond afbeelding')}
+										<span className="p-cp-settings__hint">
+											(
+											{t(
+												'Minimum {{minWidth}}px x {{minHeight}}px, max 500kb.',
+												{
+													minWidth,
+													minHeight,
+												}
+											)}
+											)
+										</span>
+									</p>
+									<div className="p-cp-settings__leeszaal-image-buttons">
+										<Button
+											label={t('Upload nieuwe afbeelding')}
+											variants="outline"
+										/>
+										<Button
+											label={t('Verwijderen')}
+											iconStart={<Icon name="trash" />}
+											variants="text"
+										/>
+									</div>
+								</div>
+							</div>
+							{newColor !== savedColor &&
+								renderCancelSaveButtons(
+									() => setNewColor(savedColor),
+									() => setSavedColor(newColor)
+								)}
+						</Box>
+					</article>
+
+					{/* Wachtzaal */}
+					<article className="p-cp-settings__content-block">
+						<h2 className="p-cp-settings__title">{t('Omschrijving wachtzaal')}</h2>
+						<Box className="p-cp-settings__box">
+							<p className="p-cp-settings__description">
+								{t(
+									'Dit is de wachtzaalomschrijving die bezoekers kunnen lezen op de detailpagina van je leeszaal. Leg uit waar je leeszaal om gaat, welke info men er kan vinden, vertel de bezoeker over je collectie.'
+								)}
+							</p>
+							<RichTextEditor
+								onChange={(state) => {
+									if (!savedWachtzaalState) {
+										setSavedWachtzaalState(state);
+									}
+									setNewWachtzaalState(state);
+								}}
+								initialHtml={
+									'<p><strong>Leeszaal 8</strong></p><p></p><h4><strong>Praktische informatie</strong></h4><p></p><p>In deze leeszaal vind je alles ...</p>'
```

Comment:
Can we make this translatable?
Then meemoo can enter the text themselves through the translations.

Reply:
This is a placeholder value for the RTE isn't it? I doubt it should be a translation

Reply:
Yes, placeholder.